### PR TITLE
Notebookbar Tab Content vertical-align

### PR DESCRIPTION
**Pull request by Andreas-Kainz**
Signed-off-by: andreas kainz <kainz.a@gmail.com>
Change-Id: Iaf876e786e8bc24dc6c03c412d12dc5e4700119c

### CODE REVIEW

**Comment by Andreas-Kainz** [comment]
### Alignment master
Screenshot_20210505_224807

### with Nextcloud Integration
Screenshot_20210505_224736

**Comment by Andreas-Kainz** [comment]
@pedropintosilva please review

checked with chrome and firefox on desktop.

File: loleaflet/css/notebookbar.css [hunk 1]
```diff
@@ -79,7 +80,7 @@
 	display: inline-block;
 	position: relative;
 	z-index: 1000;
-	height: 100%;
+	height: 32px;
```
Comment (pedropintosilva) [inline loleaflet/css/notebookbar.css:83]: I don't think this change is necessary. Can you please check and if it's not remove it from the commit?

**Review by pedropintosilva — CHANGES_REQUESTED** [review]
also please change the commit subject and add a message as well.:

- Subject: what the commit does in an imperative and succinct way (minimal summary and never ending with a period). Example: `Notebookbar: Tab: Fix vertical alginment`
- Blank line
- Message: 
  - What it fixes (present tense)
  - If the change is very simple conceptually, that one line might be enough
  -  Explain the problem being solved by this commit. More importantly, '''explain ''why'' these changes are being made''', as opposed to ''how''
  - Example: 
```
Center align tab titles (vertically) while removing when
possible hard-coded measurements from nearby elements:

- element a
- element b
- element c

Remove unnecessary padding from cyx
```

File: loleaflet/css/notebookbar.css (hunk 1 above)
Comment (Andreas-Kainz) [reply loleaflet/css/notebookbar.css]: checked and isn't necessary.

**Pull request merged**